substratevm/src/com.oracle.svm.core.genscavenge/src/com/oracle/svm/core/genscavenge/HeapPolicy.java

```diff
@@ -186,6 +186,31 @@ private static UnsignedWord getAllocationBeforePhysicalMemorySize() {
         return WordFactory.unsigned(HeapPolicyOptions.AllocationBeforePhysicalMemorySize.getValue());
     }
 
+    /* Survivor space configuration */
+
+    public static boolean useSurvivors() {
```

Comment:
I do not see a reason to keep the "old" mode without survivor spaces. Do you have benchmarks or use cases where performance is better without survivor spaces?

Reply:
done, we keep the "old" mode because we want to offer a switchable option, there is no regression of the "old" mode in our test